src/routes/plans/[id]/+page.svelte

```diff
@@ -109,9 +109,16 @@
   $: if ($plan && $simulationDataset !== undefined) {
     if ($simulationDataset !== null) {
       const datasetId = $simulationDataset.dataset_id;
-      effects
-        .getResources(datasetId, $plan.start_time, $plan.duration)
-        .then(newResources => ($resources = newResources));
+      let startYmd = $plan.start_time;
```

Comment:
🥷
```ts
const startYmd = $simulationDataset?.simulation_start_time ?? $plan.start_time
```

Duration calc is not as clean unfortunately, although you could 1-line it?

Reply:
Yeah can one line it, not too bad.